code/modules/admin/player_panel.dm

```diff
@@ -517,6 +517,21 @@
 					dat += "<td><A href='?priv_msg=[traitor.key]'>PM</A></td></tr>"
 			dat += "</table>"
 
+		if(SSticker.mode.brother_teams.len > 0)
+			dat += "<br><table cellspacing=5><tr><td><B>Brothers</B></td><td></td><td></td></tr>"
+			for(var/datum/objective_team/brother_team/team in SSticker.mode.brother_teams)
+				for(var/datum/mind/brother in team.members)
+					var/mob/M = brother.current
+					if(M)
+						dat += "<tr><td><a href='?_src_=holder;adminplayeropts=\ref[M]'>[M.real_name]</a>[M.client ? "" : " <i>(No Client)</i>"][M.stat == DEAD ? " <b><font color=red>(DEAD)</font></b>" : ""]</td>"
+						dat += "<td><A href='?priv_msg=[M.ckey]'>PM</A></td>"

```

Comment:
you copypasted this code from somewhere, check if there are no existing admin macros for each of these, because otherwise they need security hrefs (see @Cyberboss's pr on hrefs)